src/apps/competitions/tests/unpacker_test_data.py

```diff
@@ -200,6 +200,39 @@
     }
 ]
 
+PHASES2 = [
```

Comment:
I hate this solution, but i wasn't sure what the best way to handle the fact that Phases for v2 has `hide_output` and v1.5 does not. There is probably a better solution than this, I just can't think of it.